.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: Release
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The git tag for the version to use for index-demo.html
+        required: true
+env:
+  BUCKET: models-resources
+  PREFIX: nass-plugin
+  SRC_FILE: index-top.html
+  DEST_FILE: index-demo.html
```

Comment:
this should be `index.html`

Reply:
@dougmartin There isn't an index.html file in the dist folder - but there is a demo.html file. Do you think this should be changed to demo-top.html and demo.html?

Reply:
No, the `DEST_FILE` is what is created using the contents of `SRC_FILE` when the release is made.  It needs to be `index.html` -- you can see the same code for AP here:

https://github.com/concord-consortium/activity-player/blob/master/.github/workflows/release_production.yml#L12

As is, this PR won't copy over the demo code.  I think it is fine as is so we can get a release out but we should come back and add code using the AWS cli to handle the demo.

Reply:
Got it, thank you!